centos-ci/jobs/gluster-kubernetes.yml

```diff
@@ -0,0 +1,44 @@
+- job:
+    name: gluster_gk-deploy
```

Comment:
This would be unwise, as the gluster-kubernetes repo could grow to encompass more than just gk-deploy. I'd go with just `gluster_kubernetes`.